library/java/src/io/envoyproxy/envoymobile/engine/AndroidEngineImpl.java

```diff
@@ -60,6 +60,16 @@ public int recordGaugeSub(String elements, int amount) {
     return envoyEngine.recordGaugeSub(elements, amount);
   }
 
+  @Override
+  public int recordGaugeSub(String elements, int amount) {
+    return envoyEngine.recordGaugeSub(elements, amount);
+  }
+
+  @Override
+  public int recordHistogramDurationMs(String elements, int duration) {
```

Comment:
This will be used in a follow-up PR for the actual interface that clients will use to record histogram stats.